R/post-body.R

```diff
@@ -1,34 +1,47 @@
 postBodyFilter <- function(req){
   handled <- req$.internal$postBodyHandled
-  if (is.null(handled) || handled != TRUE){
-    body <- paste0(req$rook.input$read_lines(), collapse = "\n")
-    charset <- getCharacterSet(req$HTTP_CONTENT_TYPE)
-    args <- parseBody(body, charset)
-    req$postBody <- body
+  if (is.null(handled) || handled != TRUE) {
+    body <- req$rook.input$read()
+    type <- req$HTTP_CONTENT_TYPE
+    args <- parseBody(body, type)
+    req$postBody <- body #this is a raw body contrary to what plumber did before
```

Comment:
Need to maintain the original behavior of `postBody`

Could get around this by allowing for a global option

```r
if (getOption("plumber.postBody", TRUE)) {
	req$postBody <- paste0(req$rook.input$read_lines(), collapse = "\n")
}
req$rawPostBody <- body # req$rook.input$read() # this object
```